libs/sdk-ui-kit/src/SettingWidget/StylingSettingWidget/StylingSettingWidget.tsx

```diff
@@ -34,7 +34,10 @@ export interface IStylingSettingWidgetProps<T> {
     footerHelpTitle?: string;
     footerMobileMessage?: string;
     className?: string;
+    disabledCancelButton?: boolean;
+    disabledApplyButton?: boolean;
```

Comment:
Should we rename `disabledApplyButton` → `shouldDisableApplyButton` for the prop to be somewhat more self-descriptive and more grammatically correct?

Reply:
I fixed all comments. Please check again. Thanks.